demos/drawer/persistent-drawer.html

```diff
@@ -52,16 +53,21 @@
 
       .demo-main {
         padding-left: 16px;
+        padding-bottom: 16px;
+      }
+
+      .demo-main__control {
```

Comment:
1. Rename to `demo-switch`
2. Move this class to `demos/drawer/drawer.scss`
3. File a follow-up chore in Pivotal to consolidate and move existing common/shared styles out of the `<style>` blocks in each individual drawer demo page into `demos/drawer/drawer.scss`

Ditto for other files